dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/sanitizer/ShortUriGitRemoteUrlSanitizer.kt

```diff
@@ -0,0 +1,27 @@
+package com.datadog.gradle.plugin.internal.sanitizer
+
+import java.net.URI
+
+internal class ShortUriGitRemoteUrlSanitizer : UrlSanitizer {
```

Comment:
It might be a good idea to let this extend the `FullUriGitRemoteUrlSanitizer`. If at some point we need to change the sanitization rule, you could reuse the same logic and only have the change done in one place: 

```
override fun sanitize(url: String): String {
    val sshUri ="$SSH_SCHEMA_PREFIX$url"
    val sanitizedUri = super.sanitize(sshUri)
    return sanitizedUri.removePrefix(SSH_SCHEMA_PREFIX)
}
```